### PR TITLE
0.2

### PR DESCRIPTION
**Pull request by clemblanco**
Supports PHP:
- 8.0

Supports Laravel:
- 7.x
- 8.x

### CODE REVIEW

**Review by oaklees — APPROVED** [review]
How do we maintain 0.1.x patches after this merge to master? Will that remain back on its own 0.1 branch instead, with tags 0.1.1/2/3 etc as it progresses, leaving the latest 0.2 here to progress on master? Not sure what the best practices are around this.. I presume the latest and greatest progresses on master though as makes sense.

File: composer.json [hunk 1]
```diff
@@ -32,6 +31,9 @@
         }
     },
     "extra": {
+        "branch-alias": {
+            "dev-master": "0.2-dev"
```
[the remark this answers is not in the document]
Comment (clemblanco) [reply composer.json:35]: Yes so the idea is to use `master` for the latest version of that package, compatible with the latest versions of PHP and Laravel.

This means that `0.1.x` will be for Laravel 5 and 6 and `0.2.x` will be for Laravel 7 and 8.

Having that alias avoids having to keep both `master` and `0.2` branches and just keep `master` for latest version.

File: src/Concerns/Exportable.php [hunk 2]
```diff
@@ -15,7 +14,7 @@ trait Exportable
     /**
      * @inheritDoc
      */
-    public function send(MailerContract $mailer)
+    public function send($mailer)
```
[the remark this answers is not in the document]
Comment (clemblanco) [reply src/Concerns/Exportable.php:17]: Yeah I know but it's in the core so we kinda have to adapt 🤷‍♂️

**Comment by clemblanco** [comment]
> How do we maintain 0.1.x patches after this merge to master? Will that remain back on its own 0.1 branch instead, with tags 0.1.1/2/3 etc as it progresses, leaving the latest 0.2 here to progress on master? Not sure what the best practices are around this.. I presume the latest and greatest progresses on master though as makes sense.

Yeah that was my idea. I looked at `laravel-debug-bar` and other packages and it seems to be like how they do things.

We would have `0.1.1`, `0.1.2`, `0.1.3`, `0.1.4`... for Laravel 5+6 until we reach a stable version and have `1.0.0`, `1.0.1`, `1.1.0`, `1.1.1`... for Laravel 5+6

Same thing for `0.2` and `2.x.x` for Laravel 7+8.

Did you have something else in mind?

**Pull request merged**